WebService/Auth/AuthMiddleware.cs

```diff
@@ -103,6 +107,9 @@ public Task Invoke(HttpContext context)
             var header = string.Empty;
             var token = string.Empty;
 
+            // Store this setting to skip validating authorization in the controller if enabled
+            context.Request.SetAuthRequired(this.config.AuthRequired);
```

Comment:
Auth is pretty vague.  Does it refer to authentication or authorization?  This is a general comment.  So if we refer to both authentication and authorization using "auth" then we need to start spelling it out in all cases for clarity... or use different words.

Reply:
I think we are pretty deep on using "auth" at this point, so it is out of scope of this PR